### ISSUE TITLE
Inconsistent MultiverseIds for ZNR Extended Art Cards

### ISSUE THREAD

**Issue by AEFeinstein** [mtgjson/mtgjson#718]
#### Version(s) affected
- [ ] MTGJSON 4
- [X] MTGJSON 5

#### Version(s) tested against
v5.0.1+20201117

#### Description of Bug

Multiverse IDs for ZNR extended art cards seem to be inconsistent.

For instance, "Nissa of Shadowed Boughs" has `multiverseId`s for both #231 and #283.

But "Agadeem's Awakening // Agadeem, the Undercrypt" only has `multiverseId`s for #90a/#90b, not #335a/#335b
| Card Number | `multiverseId` | Missing |
|-|-|-|
|90a|491723| |
|90b|491724| |
|335a|495619|missing|
|336b|495620|missing|

There are likely more cards affected, I didn't do an exhaustive search for multiverseId inconsistency.

**Comment by AEFeinstein** [comment]
Likely a duplicate of #714, I didn't check other open tickets first 😢

**Comment by ZeldaZach** [comment]
Looks to be an issue upstream

**Comment by ZeldaZach** [comment]
I've submitted a fix for this upstream. Will wait on it!

**Comment by AEFeinstein** [comment]
Thanks!

**Comment by ZeldaZach** [comment]
Resolved!

**Comment by AEFeinstein** [comment]
Thanks again!
